src/flb_trace_chunk.c

```diff
@@ -0,0 +1,654 @@
+#include <fcntl.h>
+
```

Comment:
always include `<fluent-bit/flb_info.h>` header as the first one, since it defines some pre-processor macros that other headers might check